CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ set(MODULES_SOURCES
   ${CMAKE_SOURCE_DIR}/src/RunningContext.cpp
   ${CMAKE_SOURCE_DIR}/src/ConfigurationParser.cpp
   ${CMAKE_SOURCE_DIR}/src/ModulesFactory.cpp
+  ${CMAKE_SOURCE_DIR}/src/OpportunisticResourceEstimator.cpp
```

Comment:
It should be named ResourceEstimator if nothing in its implementation is opportunistic

Reply:
Good point i have renamed it CommandResourceEstimator to avoid confusion with **mesos::ResourceEstimator**